Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/exceptions.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Exceptions that can occur while interacting with a Launcher
+"""
+
+
+class LyTestToolsFrameworkException(Exception):
+    """ Indicates that an Exception resulted from inside the LyTestTools framework """
+
+    def __init__(self, inner_exception=None):
+        self.inner_exception = inner_exception
+
+    def __str__(self):
+        return f"This exception was generated from inside the LyTestTools framework. The inner exception is {self.inner_exception}"
```

Comment:
How does the clarify the exception? It's also potentially confusing when the inner exception is empty.

I don't quite see a need to override the default how exceptions are already handled when using `raise x from y`